packages/aws-cdk-lib/aws-codebuild/lib/project.ts

```diff
@@ -2175,3 +2231,13 @@ export enum ProjectNotificationEvents {
 function isBindableBuildImage(x: unknown): x is IBindableBuildImage {
   return typeof x === 'object' && !!x && !!(x as any).bind;
 }
+
+export function isLambdaComputeType(computeType: ComputeType): boolean {
+  return [
+    ComputeType.LAMBDA_1GB,
+    ComputeType.LAMBDA_2GB,
+    ComputeType.LAMBDA_4GB,
+    ComputeType.LAMBDA_8GB,
+    ComputeType.LAMBDA_10GB,
+  ].includes(computeType);
```

Comment:
We're going to have to continue to maintain this list every time we add a new Lambda compute type. Ideally we'd have this be an enum-like class but can't change that now. The only thing I can think of here is that instead of maintaining a list, we commit to having the `BUILD_LAMBDA` prefix for all Lambda Compute types and then check for that in this function.